src/components/App.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+import Page from './Page';
+
+export default function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
+
+  function handleClick(value) {
+    setState({
+      count: count + value,
+    });
```

Comment:
`state` 객체안에 속성이 하나 더 있다면 어떤 일이 벌어질까요?
상태를 변경해줄 때 객체나 배열을 사용하시면 기존 상태를 추가해주셔야 해요.


```suggestion
    setState({
      ...state,
      count: count + value,
    });
```

Reply:
다른 속성이 있는 경우에 덮어버리는군요 🤭  감사합니다!